src/contracts/actions-builders/CappedTokenTransfers.sol

```diff
@@ -27,19 +28,15 @@ contract CappedTokenTransfers is ICappedTokenTransfers, ActionHubChild, ActionsB
 
   /**
    * @notice Constructor that sets up the token, amount and recipient
-   * @param _parent The parent that deployed the actions builder. This is the factory address.
    * @param _token The token contract address
    * @param _amount The amount of tokens to transfer
    * @param _recipient The recipient of the tokens
-   * @param _actionHub The parent hub of the action. In this case, it's the CappedTokenTransfersHub contract that created this action Builder. It is the same as _parent when the action builder is correctly created by a hub.
    */
   constructor(
-    address _parent,
     address _token,
     uint256 _amount,
-    address _recipient,
-    address _actionHub
-  ) ActionsBuilder(_parent) ActionHubChild(_actionHub) {
+    address _recipient
+  ) ActionsBuilder(msg.sender) ActionHubChild(msg.sender) {
```

Comment:
i'm a bit confused about why this gets `msg.sender`, or the logic in general about parents and actions - the only thing i can infer is that this ties the parent to `msg.sender` because it will come from a `hub`, but the actions come from factories and don't enforce the parent, and those without factory use address(0), is there any reason why this is like this?